ansible/idr-mineotaur.yml

```diff
@@ -1,28 +1,45 @@
+# Load hostvars
+- hosts: "{{ idr_environment | default('idr') }}-omero-hosts"
+
 # Setup IDR Mineotaur server
 - hosts: "{{ idr_environment | default('idr') }}-a-dockermanager-hosts"
 
   tasks:
 
+  - name: Get rsync IP
+    set_fact:
+      omero_rsync_host_ansible : >-
```

Comment:
My hope is that you would have a suggestion how we want to let `-a-dockermanger`s download from somewhere else. One option, e.g., would be to have `-a-omero`s _also_ serve rsync.